exist-core/src/main/java/org/exist/xquery/value/BooleanValue.java

```diff
@@ -114,14 +114,17 @@ public boolean compareTo(Collator collator, Comparison operator, AtomicValue oth
     }
 
     public int compareTo(Collator collator, AtomicValue other) throws XPathException {
```

Comment:
Some more missing `final` tags

Reply:
Hmm... this isn't related to my changes. I will add a commit to mark it as a final as you have requested